asistente_ladm_col/gui/data_model_converter/dlg_data_model_converter.py

```diff
@@ -0,0 +1,495 @@
+                          gcarrillo@linuxmail.org
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtWidgets import (QDialog,
+                                 QMessageBox,
+                                 QDialogButtonBox,
+                                 QSizePolicy)
+from qgis.PyQt.QtCore import (Qt,
+                              QSettings,
+                              QCoreApplication,
+                              pyqtSignal)
+from qgis.PyQt.QtGui import QColor
+from qgis.core import (Qgis,
+                       QgsProcessingException,
+                       QgsProcessingFeedback,
+                       QgsProject)
+from qgis.gui import (QgsMessageBar,
+                      QgsMapCanvas)
+
+import processing
+
+from asistente_ladm_col.config.general_config import (SUPPLIES_DB_SOURCE,
+                                                      COLLECTED_DB_SOURCE,
+                                                      SETTINGS_CONNECTION_TAB_INDEX)
+from asistente_ladm_col.config.keys.common import REQUIRED_MODELS
+from asistente_ladm_col.config.ladm_names import LADMNames
+from asistente_ladm_col.app_interface import AppInterface
+from asistente_ladm_col.lib.logger import Logger
+from asistente_ladm_col.lib.context import Context
+from asistente_ladm_col.gui.dialogs.dlg_settings import SettingsDialog
+from asistente_ladm_col.utils import get_ui_class
+
+DIALOG_DATA_MODEL_CONVERTER_UI = get_ui_class('data_model_converter/dlg_data_model_converter.ui')
+
+
+class DataModelConverterDialog(QDialog, DIALOG_DATA_MODEL_CONVERTER_UI):
+    #on_result = pyqtSignal(bool)  # whether the tool was run successfully or not
+
+    def __init__(self, conn_manager=None, parent=None):
+        QDialog.__init__(self, parent)
+        self.setupUi(self)
+        self.parent = parent
+        self.conn_manager = conn_manager
+        self.logger = Logger()
+        self.app = AppInterface()
+
+        self._running_tool = False        
+        self.tool_name = QCoreApplication.translate("DataModelConverterDialog", "Data Model Converter")
+ 
+        # we will use a unique instance of setting dialog
+        self.settings_dialog = SettingsDialog(self.conn_manager, parent=parent)
+        # The database configuration is saved if it becomes necessary
+        # to restore the configuration when the user rejects the dialog
+        self.init_db_converted = None
```

Comment:
What do you think if we use source and target instead of pre_converted and converted? 
It is something very simple but I think it gives clarity to the code.

Reply:
Agreed and fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/4888b2f347196a8dc65bad715dc7f8c71a5731dd